configure

```diff
@@ -303,11 +303,11 @@ if args.comp == 'icc':
     else:          f.write("CXX_COMPILE_FLAGS += -Ofast -ipo\n") # -inline-forceinline
 
 #
-# MPICH
+# MPI
```

Comment:
Please leave specific to MPICH. Only mpich or mvapich are supported (i.e. not openmpi). This is true of amrex as well. The documentation should reflect the mpich dependency.

Reply:
Hi @bsrunnels!

This isn't true of amrex? Why wouldn't we expand to more/all MPI libraries?

Reply:
@nmnobre I agree that it is ideal to support all, but parallelization does not always work correctly when compiled with openmpi. A while back (several years ago) we concluded that openmpi was to blame and dropped alamo support for it. 
I don't think that there's anything preventing you using openmpi (i.e. if you have that installed instead of mpich then it should use it automatically), but keeping mpich specified in the docs is helpful to encourage use of mpich and avoid future errors.